### PR TITLE
added homework 3 Hlib Sokolovskyi

### PR DESCRIPTION
**Pull request by Desferatum**

### CODE REVIEW

**Comment by Desferatum** [comment]
modified repository for merging, but one folder doesn't pull from main (KasianenkoTymur)

File: HlibSokolovskyi/HW_3/1.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Now is better than never.
+Although never is often better than *right* now.
+If the implementation is hard to explain, it's a bad idea.
+If the implementation is easy to explain, it may be a good idea.
+Namespaces are one honking great idea -- let's do more of those!"""
+
+better_word = "better"
+never_word = "never"
+is_word = "is"
+
```
Comment (kolyasalubov) [inline HlibSokolovskyi/HW_3/1.py:29]: good job

File: HlibSokolovskyi/HW_3/2.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+2.1
+
```
Comment (kolyasalubov) [inline HlibSokolovskyi/HW_3/2.py:3]: good job

File: HlibSokolovskyi/HW_3/3.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (kolyasalubov) [inline HlibSokolovskyi/HW_3/3.py:1]: good job

**Review by kolyasalubov — APPROVED** [review]

**Pull request merged**